data/mappings/keyboard_aliases.hjson

```diff
@@ -200,6 +200,9 @@
     "gmmk/pro/iso": {
         "target": "gmmk/pro/rev1/iso"
     },
+    "handwired/ergo_s-1":{
+        "target": "handwired/dactyl_cc"
+    }
```

Comment:
```suggestion
```

Reply:
Can I ask as to what purpose for this change?

Would it be preferred the Ergo S-1 has it's own dir in `keyboards`?

Reply:
This is not what `keyboard_aliases` is for. Hyphens are also not allowed in keyboard names.

QMK does not differentiate keyboards by things like housing; per the readme, these two boards are "electronically the same", which I assume to mean it is the exact same PCB. Therefore it should have only one name in the repo.